examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
> that function signature is required for windows

for what compiler/generator? I've just tried simple example with `char** argv` and it works fine with Visual Studio 2013.